code/game/turfs/simulated/floor/plating.dm

```diff
@@ -53,6 +55,8 @@
 			if(!W.use(1))
 				return
 			var/turf/open/floor/T = ChangeTurf(W.turf_type)
+			if(W.HAS_SECONDARY_FLAG(src, FROZEN))
+				T.MakeSlippery(TURF_WET_PERMAFROST)
```

Comment:
?
yes there is an issue where it can be too warm for the new ice tile(because frozen items don't unfreeze naturally) but that's not _my_ problem, it's some goof thing.